Source/WebKit/UIProcess/API/Cocoa/WKWebpagePreferencesPrivate.h

```diff
@@ -74,7 +74,6 @@ typedef NS_OPTIONS(NSUInteger, _WKWebsiteColorSchemePreference) {
     _WKWebsiteColorSchemePreferenceDark,
 } WK_API_AVAILABLE(macos(13.0), ios(16.0));
 
-// Allow controlling a per-page network connection integrity policy.
 typedef NS_OPTIONS(NSUInteger, _WKWebsiteNetworkConnectionIntegrityPolicy) {
```

Comment:
Should we follow up this PR with another one that adds new SPI with the correct naming scheme?

Reply:
Yep! Filed <https://webkit.org/b/257725> for this.